pages/vi/vi-github-and-repositories.md

```diff
@@ -12,7 +12,7 @@ On GitHub, our software code is organized in repositories, each of which represe
 
 As previously mentioned, you fork a repository to work on your own GitHub account, and then send back your changes to the upstream repository in the form of a pull request (PR). You went through this process to fork open-learning-exchange.github.io into your own repository and rename it to your own username. Normally, we do not rename repositories, but Markdown Wikis are slightly different. In the future, plan on forking different repositories, working on them, then sending your work to the upstream repo via PR. The general GitHub structure is outlined below.
 
-![Repository Relationships](images/vi-repo-diagram.png)
```

Comment:
I think the original is correct here. Repository Relationships

Reply:
The *original* original text was actually "Repositories Relationship" which @ajGingrich believed was more accurate than "Repository Relationships." I personally agree with you and think "Repository Relationships" is better.

Reply:
If you both think it's better for repository relationships, I'm fine with that.